pend_task.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require_once('config.php');
+
+$user_id = $_POST['userid'];
+
+$sql = "UPDATE data SET status=1 WHERE id=".$user_id;
```

Comment:
@Dlewis89 You have done a great job. I ran the code locally and it worked without any issues...

---
It'll be great if you could help me fix a small underlying concern. The **Pending Tasks** are to be completed later. They are just marked pending. So, how about the pending tasks still remain on the list with a different thumbnail image _(so that we can have an idea about the pending ones...)_. They must only disappear after they are marked as completed.

Reply:
Glad to hear it, and sure I'll be more than happy to help out. I'll take a look at it tonight. Can you open a different issue for this concern since the previous one is closed and I'll work off of the new one. Thank you.